src/lib/ui/Spinner/Spinner.tsx

```diff
@@ -0,0 +1,9 @@
+import classNames from 'classnames'
+
+interface SpinnerProps {
+  size?: 'small' | 'medium' | 'large'
+}
```

Comment:
We should make it required
```suggestion
interface SpinnerProps {
  size: 'small' | 'medium' | 'large'
}
```

Or set a default
```ts
 Spinner({ size = 'small' } = {}: SpinnerProps)
```

Reply:
Ahh thanks—will add default